python/lsst/afw/image/calib.cc

```diff
@@ -19,60 +19,100 @@
  * the GNU General Public License along with this program.  If not, 
  * see <https://www.lsstcorp.org/LegalNotices/>.
  */
+#include <memory>
+#include <vector>
 
 #include <pybind11/pybind11.h>
 #include <pybind11/operators.h>
 #include <pybind11/stl.h>
 
+#include "numpy/arrayobject.h"
+#include "ndarray/pybind11.h"
+#include "ndarray/converter.h"
+#include "ndarray_fwd.h"
+
+#include "lsst/base.h"  // for CONST_PTR
```

Comment:
I will go through the pybind11 .cc files that I touched on this ticket and eliminate `PTR`, `CONST_PTR` and `#include "lsst/base.h"`.